tests/trainer/test_lr_finder.py

```diff
@@ -262,3 +264,12 @@ def test_suggestion_with_non_finite_values(tmpdir):
 
     assert before_lr == after_lr, \
         'Learning rate was altered because of non-finite loss values'
+
+
+def test_lr_finder_fails_fast_on_bad_config(tmpdir):
+    """ Test that tune fails if the model does not have a lr BEFORE running lr find """
+    # note: this did not raise an exception before #5648 because lr_find is skipped
+    # during fast_dev_run and the lr attribute check was done after lr_find
```

Comment:
#5648 is an issue, how does it relate here?

Reply:
typo, meant to reference this PR

```suggestion
    # note: this did not raise an exception before #5638 because lr_find is skipped
    # during fast_dev_run and the lr attribute check was done after lr_find
```